### DISCUSSION TITLE
returning letterboxing to stable

### DISCUSSION THREAD

**Discussion by elimango** [ppy/osu#26673] [Ideas]
I'm wondering about the letterboxing in lazer, it seems like it's handled a completely different way. I've tried the exact same letterboxing on lazer as I use on stable, but lazer feels incredibly different. I'm wondering if there are plans to revert letterboxing back to the way it's handled in stable.

**Comment by peppy** [comment]
a good start would be to explain what feels different

**Reply by elimango** [reply]
For sure. As im posting this for a friend, please bear with me and let me know if something doesn't make sense;
When letterboxing in lazer, my area feels completely different. It feels as if my area is being mapped to the entire tab, including the outer border, rather than just the smaller window. Whereas in stable, it seems like it maps my area to the smaller window and not the outer border.

**Reply by peppy** [reply]
And they are using "everything" scaling?

**Reply by elimango** [reply]
If referring to screen scaling; yes, they've confirmed it's set to everything.

**Reply by peppy** [reply]
Tracking at https://github.com/ppy/osu-framework/issues/6150.
